generate_from_simbev.py

```diff
@@ -254,9 +254,14 @@ def datetime_from_timestep(timestep):
                 # general sanity checks
                 simbev_soc_start = float(row["SoC_start"])
                 simbev_soc_end = float(row["SoC_end"])
-                assert simbev_soc_start > 0 and simbev_soc_end > 0, \
+                assert simbev_soc_start >= 0 and simbev_soc_end >= 0, \
                     "SimBEV created negative SoC for {} in row {}".format(
                         vehicle_name, idx + 2)
+                if simbev_soc_start == 0:
+                    print("WARNING: SoC_start == 0 in row {}".format(idx + 2))
+                if simbev_soc_end == 0:
+                    print("WARNING: SoC_end == 0 in row {}".format(idx + 2))
+
```

Comment:
The problem is that SoCs are saved as floating point numbers (rounded to two decimal places, but still). I would argue an SoC of 1% is still unrealistic, but this can be up to debate. My suggestion:
```python
soc_threshold = vars(args).get(MIN_SOC_THRESHOLD, 0.05)
if simbev_soc_start <  soc_threshold or simbev_soc_end < soc_threshold:      
        print("WARNING: SimBEV created very low SoC for {} in row {}".format(vehicle_name, idx + 2))     
```
This way, the threshold can be set from a config file.

Reply:
:+1:
You gonna add it? (I'm not familiar with the cfg stuff and CoC so I dunno what else needs to be adjusted - cfg files, docs etc.)